### DISCUSSION TITLE
Save Space on Device Feature

### DISCUSSION THREAD

**Discussion by FabryXY** [savvasdalkitsis/uhuruphotos-android#1299] [Q&A]
Hi all,

Thank you for your work, the app is very promising and I am planning to test it a little further with my librephoto server and eventually ditch google photo. 
I have just a question about device space. Is it possible to do something similar to the stock google photo app, keeping the actual full quality photo on the server and just a small thumbnail on the app side?

Thank you

Fab

**Comment by savvasdalkitsis** [comment]
Hello and thanks for the kind words.

This was discussed here: https://github.com/savvasdalkitsis/uhuruphotos-android/issues/900

**Comment by FabryXY** [comment]
Thank you so much 😊
